sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy

```diff
@@ -124,8 +125,7 @@ class ContainerAPITest extends APISpec {
         response.getValue().getLeaseState() == LeaseStateType.AVAILABLE
         response.getValue().getLeaseStatus() == LeaseStatusType.UNLOCKED
         response.getValue().getMetadata().size() == 0
-        response.getValue().getDefaultEncryptionScope()
-        response.getValue().hasDenyEncryptionScopeOverride() != null
+        response.getValue().getBlobContainerEncryptionScope()
```

Comment:
This returns an object, right? I think you need to check the values on the object.

Reply:
done